other/materials_designer/specific_examples/defect_point_adatom_island_mos2_pt.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    MATERIAL_NAME = "MoS2.*2D"  # Name of the material
</pre><p>MATERIAL_NAME_REGEX</p><p> </p><p>Plus we should mention that it is used to get material from mat3ra-standata</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>